lms/djangoapps/support/views/program_enrollments.py

```diff
@@ -71,48 +76,36 @@ def post(self, request):
             }
         )
 
-    @staticmethod
-    def _validate_and_link(program_uuid_string, linkage_text):
+
+class LinkProgramEnrollmentSupportAPIView(GenericAPIView):
+    """
+    Support-only API View for changing learner enrollments by support
+    staff
+    """
+    authentication_classes = (
+        JwtAuthentication, SessionAuthentication
+    )
+    permission_classes = (IsAuthenticated,)
+
+    @method_decorator(require_support_permission)
+    def post(self, request):
         """
-        Validate arguments, and if valid, call `link_program_enrollments`.
+        Creates and changes learner enrollments by support staff
 
-        Returns: (successes, errors)
-            where successes and errors are both list[str]
+        * Example Request:
+            - POST /support/link_program_enrollments_details/
```

Comment:
Add a sample body for the POST request and a sample response.

Reply:
Updated.